components/treasuryV2/Details/RealmAuthorityDetails/Config.tsx

```diff
@@ -31,70 +36,8 @@ export default function Config(props: Props) {
   const { mint } = useRealm()
   const [editRealmOpen, setEditRealmOpen] = useState(false)
 
-  const config = [
```

Comment:
Better to avoid stuff like this because it will cause unnecessary re-renders and is imo just less legible and editor friendly.